integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CarbonShowCacheCommand.scala

```diff
@@ -74,7 +92,15 @@ case class CarbonShowCacheCommand(tableIdentifier: Option[TableIdentifier],
       /**
        * Assemble result for database
        */
-      getAllTablesCache(sparkSession)
+      if (showExecutorCache == false) {
```

Comment:
"==false " not required...Change in all other places also

Reply:
done